### PR TITLE
Display messages for configure,update and delete command.

### LINKED ISSUE #108: Display messages for vdoctl configure cm, update and delete command
**Issue/Feature Description:**

**What type of Issue/Feature is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind enhancement

**Why this fix/ feature is needed(give scenarios or use cases):**
Display messages for vdoctl configure cm, update and delete command, which confirms the user that the command has been executed successfully and what can be performed next.

### PR DESCRIPTION
**Pull request by ridaz**
**What type of PR is this?**
> /kind enhancement

**What this PR does / why we need it**:

Displays an appropriate message to the user after the command is invoked.
This PR will bring in the following changes :
1. vdoctl configure compatibility-matrix - Tip for the user since matrix has been configured, they can proceed with deploy VDO.
2. vdoctl delete - Message displaying since vdoctl delete has been invoked, the deployments will be deleted.
3. vdoctl update-matrix - Message displaying matrix has been updated successfully.

**Which issue(s) this PR fixes**:

Fixes #108 

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
/kind TESTED

**Test Report**:

Tested the following flow:

```
bin/vdoctl deploy --spec https://raw.githubusercontent.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/main/artifacts/vanilla/vdo-spec.yaml
Tip: now that you have deployed VDO, you might want to try 'vdoctl configure drivers' to configure vsphere drivers
```

```
kubectl get pods -A
NAMESPACE           NAME                                      READY   STATUS                       RESTARTS   AGE
vmware-system-vdo   vdo-controller-manager-6576ff6b8c-nk4g2   1/2     CreateContainerConfigError   0          112s
```

```
bin/vdoctl configure cm 
✔ Web URL
Web URL https://raw.githubusercontent.com/ridaz/Sample-files/main/test-matrix.yaml
Tip: Compatibility matrix has been configured, you might want to try 'vdoctl configure drivers' to configure vsphere drivers.
```

```
bin/vdoctl configure drivers
Do you want to configure CloudProvider? (Y/N) Y
Please provide the vcenter IP for configuring CloudProvider 
VC IP 10.187.159.221
Do you want to establish a secure connection? (Y/N) N
Please provide the credentials for configuring CloudProvider
Username administrator@vsphere.local
Password ********
Datacenter(s) Datacenter
Do you want to configure another vcenter for CloudProvider? (Y/N) N
Do you want to configure zones/regions for CloudProvider? (Y/N) N
You have now completed configuration of CloudProvider. We will now proceed to configure StorageProvider.
Please provide the credentials for configuring StorageProvider
Username administrator@vsphere.local
Password ********
Datacenter(s) Datacenter
Do you wish to provide custom kubelet Path? (Y/N) N
Do you wish to configure File Volumes? (Y/N) N
Thanks for configuring VDO. The drivers will now be installed.
You can check the status for drivers using `vdoctl status`
```

 ```
bin/vdoctl version
kubernetes Version : 1.22
VDO Version        : 0.1.0
vSphere Versions   : [7.0.3]
CSI Version        : 2.2.2
CPI Version        : 1.22.0

bin/vdoctl status 
CloudProvider   : Configured
	 vCenter : 
		10.187.159.221  (Credentials Verified)
	 Nodes : 
		 master-vm : ready 
		 worker-vm : ready 
StorageProvider : Deployed
	 vCenter : 
		10.187.159.221  (Credentials Verified)
```

```
NAMESPACE     NAME                                      READY   STATUS    RESTARTS   AGE
kube-system         vsphere-cloud-controller-manager-c7zdb    1/1     Running   0          3m
kube-system         vsphere-csi-controller-7f7dc49657-qb7kr   6/6     Running   0          2m53s
kube-system         vsphere-csi-node-6ssk6                    3/3     Running   0          2m53s
kube-system         vsphere-csi-node-h9zg8                    3/3     Running   0          2m53s
vmware-system-vdo   vdo-controller-manager-6576ff6b8c-nk4g2   2/2     Running   0          8m15s

```

```
bin/vdoctl update compatibility-matrix https://raw.githubusercontent.com/ridaz/Sample-files/main/matrix.yaml
Compatibility matrix has been updated successfully.
```

```
bin/vdoctl delete vdo
This command deletes the VDO operator deployment eventually the operator pods will be deleted

```

```
kubectl get pods -A
NAMESPACE     NAME                                      READY   STATUS    RESTARTS   AGE
kube-system   vsphere-cloud-controller-manager-c7zdb    1/1     Running   0          3m28s
kube-system   vsphere-csi-controller-7f7dc49657-qb7kr   6/6     Running   0          3m21s
kube-system   vsphere-csi-node-6ssk6                    3/3     Running   0          3m21s
kube-system   vsphere-csi-node-h9zg8                    3/3     Running   0          3m21s
```

**Special notes for your reviewer**:

### CODE REVIEW

File: vdoctl/cmd/configure_matrix.go [hunk 1]
```diff
@@ -77,6 +77,7 @@ var matrixConfigureCmd = &cobra.Command{
 			cobra.CheckErr(err)
 		}
 
+		fmt.Println("Tip: Compatibility matrix has been configured, you might want to try 'vdoctl deploy' to deploy VDO.")
```
Comment (asifdxtreme) [inline vdoctl/cmd/configure_matrix.go:80]: `Tip: Compatibility matrix has been configured, you might want to try 'vdoctl configure drivers' to configure Drivers.`

Flow is something like this
Deploy->Configure Matrix -> Configure Driver -> Check Status

**Review by asifdxtreme — APPROVED** [review]
LGTM

**Review by aggarwalta — APPROVED** [review]
LGTM, thanks!

File: vdoctl/cmd/delete.go [hunk 2]
```diff
@@ -58,6 +58,8 @@ Currently the command supports vanilla k8s cluster`,
 				cobra.CheckErr(fmt.Errorf("Error occurred when deleting VDO,  %v", err))
 			}
 		}
+
+		fmt.Println("Since you have invoked delete VDO command, eventually the deployments will be deleted.")
```
Comment (jvrahav) [inline vdoctl/cmd/delete.go:62]: lets word this as 
This command deletes the VDO operator deployment eventually the operator pods will be deleted

Comment (jvrahav) [reply vdoctl/cmd/delete.go:62]: is it possible to modify the delete command to vdoctl delete vdo

**Review by jvrahav — CHANGES_REQUESTED** [review]

**Review by jvrahav — APPROVED** [review]

**Pull request merged**